src/main/scala/com/cognite/sdk/scala/v1/resources/dataSets.scala

```diff
@@ -74,13 +74,13 @@ private[sdk] class DataSets[F[_]](val requestSession: RequestSession[F])
 }
 
 object DataSets {
-  implicit val dataSetUpdateEncoder: Encoder[DataSetUpdate] = deriveEncoder
-  implicit val dataSetFilterEncoder: Encoder[DataSetFilter] = deriveEncoder
-  implicit val dataSetFilterRequestEncoder: Encoder[FilterRequest[DataSetFilter]] = deriveEncoder
-  implicit val dataSetDecoder: Decoder[DataSet] = deriveDecoder
-  implicit val dataSetItemsDecoder: Decoder[Items[DataSet]] = deriveDecoder
-  implicit val dataSetItemsWithCursorDecoder: Decoder[ItemsWithCursor[DataSet]] = deriveDecoder
-  implicit val dataSetCreateEncoder: Encoder[DataSetCreate] = deriveEncoder
-  implicit val dataSetCreateItemsEncoder: Encoder[Items[DataSetCreate]] = deriveEncoder
-  implicit val dataSetListQueryEncoder: Encoder[DataSetQuery] = deriveEncoder
+  implicit val dataSetUpdateEncoder: Encoder[DataSetUpdate] = deriveEncoder[DataSetUpdate]
```

Comment:
We shouldn't need to add those types on the right hand side of `=`, did you get some error without it?

Reply:
No. It works !
I read it after you pointed out that there is no need to specify the type again.
I guess we should update it for all the resources.